File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/LeaveSettingsSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class LeaveSettingsSearch extends AbstractEntity {
	String roleName;
	String numOfSickCasualLeaves;
	String numOfEarnedLeaves;
	@Temporal(TemporalType.DATE)
	Date lastChange;
}

File: scholar-web/src/main/webapp/app/model/staff/leave/settings/LeaveSettingsSearch.js
Ext.define('scholar.model.staff.leave.settings.LeaveSettingsSearch', {
	extend: 'Ext.data.Model',
	fields : [  {
		name : 'id',
		type : 'int'
	},{
		name : 'roleName',
		type : 'string'
	},
	{
		name : 'numOfSickCasualLeaves',
		type : 'string'
	},
	{
		name : 'numOfEarnedLeaves',
		type : 'string'
	},		
	{
		name : 'lastChange',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ],
	
});